aerogear/mobile-docs
# Data Sync App provisioning and Example docs

## Motivation
https://issues.jboss.org/browse/AEROGEAR-8263

## What
Added documentation about provisioning `sync-app-apb` in OpenShift.

## Why
It was missing.

## How


## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Build docs locally by `bin/quick-build.sh local-site.yml`
2. Go to `Mobile Services > Data Sync > OpenShift deployment`
3. Check the text ther

## Checklist:


## Progress

- [x] How to build and publish Docker image for Data Sync
- [x] How to provision Data Sync using the image
- [x] How to use Example Data Sync docker image
